packages/lb-components/src/components/pointCloudView/hooks/useSingleBox.ts

```diff
@@ -120,23 +121,40 @@ export const useSingleBox = () => {
 
   /** PointCloud select next/prev one */
   const switchToNextBox = useCallback(
-    (sort = ESortDirection.ascend) => {
+    (sort = ESortDirection.ascend, manual = false) => {
       if (!topViewInstance || selectedIDs.length > 1) {
         return;
       }
 
+      if (pointCloudPattern !== EToolName.Rect && pointCloudPattern !== EToolName.Polygon) {
```

Comment:
May be this part of logic doesn't match the function name 'switchToNextBox'

Reply:
Updated on next commit